packages/core/src/shared/lsp/manifestResolver.ts

```diff
@@ -76,13 +84,17 @@ export class ManifestResolver {
         return manifest
     }
 
-    private async getLocalManifest(): Promise<Manifest> {
-        logger.info(`Failed to download latest "${this.lsName}" manifest. Falling back to local manifest.`)
+    private async getLocalManifest(silent: boolean = false): Promise<Manifest> {
```

Comment:
Always reporting "failed to download" here is misleading because this will happen when "etag didnt change".